libs/designsystem/src/lib/components/modal/modal-wrapper/compact/modal-compact-wrapper.component.ts

```diff
@@ -21,20 +29,23 @@ export class ModalCompactWrapperComponent {
     return this._ionPageReset;
   }
 
-  constructor(params: NavParams, injector: Injector) {
-    this.config = params.get('config');
+  constructor(private injector: Injector, private elementRef: ElementRef<HTMLElement>) {}
+
+  ngOnInit(): void {
     this.componentPropsInjector = Injector.create({
       providers: [{ provide: COMPONENT_PROPS, useValue: this.config.componentProps }],
-      parent: injector,
+      parent: this.injector,
     });
-    this.registerScrolling(this.config.modal);
   }
 
-  private registerScrolling(modal: Modal) {
-    modal.scrollToTop = () => {};
-    modal.scrollToBottom = () => {};
+  public close(data?: any) {
```

Comment:
Consider consistency with other method's access modifiers, either prefix with `public` or not.

Reply:
Good point, fixed 👍